cc/crypto/hpke/utils.cc

```diff
@@ -70,4 +70,30 @@ absl::StatusOr<std::vector<uint8_t>> GetResponseNonce(EVP_HPKE_CTX* hpke_ctx) {
   }
   return response_nonce;
 }
+
+std::vector<uint8_t> CalculateNonce(const std::vector<uint8_t>& base_nonce,
+                                    uint64_t sequence_number) {
+  std::vector<uint8_t> nonce(kAeadNonceSizeBytes);
+  // We use 8 here since sequence number is 64 bits.
+  for (size_t i = 0; i < 8; ++i) {
+    nonce[kAeadNonceSizeBytes - i - 1] = sequence_number & 0xff;
```

Comment:
Could you please add a comment, that the nonce is calculated this way because it should be big-endian?
https://www.rfc-editor.org/rfc/rfc9180.html#name-encryption-and-decryption